apps/frontend/src/components/Main/ShortcutNav.tsx

```diff
@@ -53,16 +51,16 @@ export const ShortcutCard = (props: ShortcutCardProps) => {
 
   return (
     <div onMouseEnter={() => setHover(true)} onMouseLeave={() => setHover(false)}>
-      <RouteLink href={props.to} hideUnderline plain>
+      <RouteLink href={props.to} style={{textDecoration: "none"}} hideUnderline plain>
```

Comment:
hideUnderline skal vel sørge for det samme som textdecoration none her, eller er det noe annet som endres også?

Reply:
Den skal ihvertfall det, det er sånn den funker i teamcat feks. isåfall, ikke legg på textdeco her, men fiks opp i routelink

Reply:
Jeg var redd at det ville føre til endringer andre steder også. Mener du at jeg bør bruke textdeco i Routelink, eller fikse det opp uten å bruke textdeco?

Reply:
hideUnderline er shortcut for style={{textDecoration: "none"}} 
se i teamkat